src/php-8.1-strftime.php

```diff
@@ -95,7 +95,9 @@ function strftime (string $format, $timestamp = null, ?string $locale = null) :
       //  in formatted strings.
       // To adjust for this, a custom calendar can be supplied with a cutover date arbitrarily far in the past.
       $calendar = IntlGregorianCalendar::createInstance();
-      $calendar->setGregorianChange(PHP_INT_MIN);
+      // NOTE: IntlGregorianCalendar::createInstance DOES NOT return an IntlGregorianCalendar instance when
+      // using a non-Gregorian locale (e.g. fa_IR)! In that case, setGregorianChange will not exist.
+      if (method_exists($calendar, 'setGregorianChange')) $calendar->setGregorianChange(PHP_INT_MIN);
```

Comment:
You should instead check that `$calendar instanceof IntlGregorianCalendar`.

Also, please use opening and closing brackets, following existing coding convention.

Reply:
:upside_down_face: Of course, sorry I didn't see those myself. I've adjusted accordingly, thanks!